apps/emqx/src/emqx_broker.erl

```diff
@@ -94,6 +103,17 @@
     end
 ).
 
+%% Subscription scope.
+%% When incoming message matches the subscription topic filter, scope can additionally
+%% narrow down message delivery.
+%% * Scope `any` means a subscription with this scope is interested in all messages.
+%% * Scope `qos0` means a subscription needs _only_ QoS 0 messages.
+%%   This is currently employed by durable sessions to route QoS 0 messages through
+%%   the realtime channel to avoid storing them in the durable storage.
```

Comment:
```suggestion
%%   the direct channel to avoid storing them in the durable storage.
```

Nit: I prefer term "direct", because term "realtime" is, stricly speaking, wrong: there's nothing realtime about forwarding gen_rpc:ordered_cast's via a TCP connection.